composer.json

```diff
@@ -9,9 +9,9 @@
         "nette/utils": "^3.2",
         "rector/rector-src": "dev-main",
         "nikic/php-parser": "^4.13",
-        "symfony/console": "^5.3",
-        "symfony/dependency-injection": "^5.3",
-        "symfony/finder": "^5.3",
+        "symfony/console": "6.0 as 5.4",
+        "symfony/dependency-injection": "6.0 as 5.4",
+        "symfony/finder": "6.0 as 5.4",
```

Comment:
Current blocker to remove these aliases is https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6095